app/controllers/users_controller.rb

```diff
@@ -355,25 +355,27 @@ def edit_profile
     render layout: 'without_sidebar'
   end
 
-  def validate_profile_website(profile_params)
-    uri = profile_params[:website]
+  def cleaned_profile_websites(profile_params)
+    sites = profile_params[:user_websites_attributes]
 
-    if URI.parse(uri).instance_of?(URI::Generic)
-      # URI::Generic indicates the user didn't include a protocol, so we'll add one now so that it can be
-      # parsed correctly in the view later on.
-      profile_params[:website] = "https://#{uri}"
+    sites.transform_values do |w|
+      w.merge({
+                label: w[:label].present? ? w[:label] : nil,
+                url: w[:url].present? ? w[:url] : nil
```

Comment:
```suggestion
                label: w[:label].presence,
                url: w[:url].presence
```

Reply:
`presence` is a handy little helper that does exactly this.

Reply:
Oh, nice, that one is on me - forgot `presence` exists, done

Reply:
New to me too -- thanks for the clue!